common/protob/messages-management.proto

```diff
@@ -123,6 +132,8 @@ message Features {
     optional bool busy = 41;                    // is the device busy, showing "Do not disconnect"?
     optional HomescreenFormat homescreen_format = 42;   // format of the homescreen, 1 = TOIf 144x144, 2 = jpg 240x240
     optional bool hide_passphrase_from_host = 43;   // should we hide the passphrase when it comes from host?
+    optional DeviceColor device_color = 44;          // color of the device
```

Comment:
making it an enum does not scale with future devices. I see two options: (a) int color id copied straight out of otp, Suite will lookup model + color id to display the right color
(b) string with possible marketing name of the color ("galaxy black", "apple red") and the firmware has a lookup table per model

Reply:
i think a) is preferable as it will allow adding more colors without needing to update firmware/bootloader

Reply:
(aa) if the colors can reasonably fit into [a-z] we can store ASCII letter in the OTP and represent it as a string for protobuf

Reply:
what is the upside?

Reply:
probably just better human readability in trezorctl without maintaning a lookup table ... but i guess it's better to have it there so that apps using trezorlib can show the human-readable name